indy_common/constants.py

```diff
@@ -101,7 +102,8 @@
           SCHEMA: ([NAME, VERSION, ATTR_NAMES]),
           GET_SCHEMA: ([], []),
           CLAIM_DEF: ([REF, DATA, SIGNATURE_TYPE]),
-          GET_CLAIM_DEF: ([REF, ORIGIN, SIGNATURE_TYPE])
+          GET_CLAIM_DEF: ([REF, ORIGIN, SIGNATURE_TYPE]),
+          REVOC_REG_DEF: ([REF, DATA, REQ_METADATA, SIGNATURE_TYPE])
```

Comment:
It doesn't match the design.
For now it would be sufficient to have the following fields:
- type
- tag
- credDefId
- value
where `value` is `type`-specific, and for `CL` type has the following fields:
- issuanceType
- maxCredNum
- publicKeys
- tailsHash
- tailsLocation